WebApp/autoreduce_webapp/selenium_tests/configuration.py

```diff
@@ -15,8 +15,8 @@
 
 from utils.project.structure import get_project_root
 
-CONFIG_PATH = Path(get_project_root(), "webtests/config.json")
-TEMP_CONFIG_PATH = Path(get_project_root(), "webtests/temp_config.json")
+CONFIG_PATH = Path(get_project_root(), "WebApp/autoreduce_webapp/selenium_tests/config.json")
+TEMP_CONFIG_PATH = Path(get_project_root(), "Webapp/autoreduce_webapp/selenium_tests/temp_config.json")
```

Comment:
Should use `PROJECT_ROOT` with `from utils.project.structure import PROJECT_ROOT`

Reply:
Aye, this wasn't available originally. Will add it Cheers 👍